### ISSUE TITLE
Adding (PRIVATE) to summary line of private docstrings

### ISSUE THREAD

**Issue by peterjc** [biopython/biopython#1466]
The following grep command identifies hundreds of private functions and methods where the one-line docstring summary does not end with ``(PRIVATE).`` as per current Biopython style:

```bash
$ grep --include '*.py' -rn -A 1 "def _[a-z]" Bio BioSQL | grep '"""' | grep -v "(PRIVATE)" 
```

(The grep will miss any cases using a multi-line ``def`` statement).

Note that given we intend to move from epydoc to sphinx-apidoc for building our HTML API documentation (see #1388 and http://mailman.open-bio.org/pipermail/biopython-dev/2017-November/021884.html for details), there may be a better way to handle in the long term this via Sphinx.

i.e. Assuming we tell Sphinx to include the private functions/methods in the HTML documentation, can it flag these as private in some way (other than us having the text "(PRIVATE)" in the string)?

**Comment by JaeDukSeo** [comment]
Is this already closed? I see it merged.

**Comment by peterjc** [comment]
@JaeDukSeo this issue is still open, lots of these docstrings are still missing the ``(PRIVATE)`` suffix.

A few were fixed in #1462 which was merged.

**Comment by Amrithasuresh** [comment]
@peterjc Shall I do the `Bio/Phylo/` directory first? Shall I give pull request for individual files? I will add `(PRIVATE)` at the end of the docstrings wherever required. 

Thank you.

**Comment by peterjc** [comment]
@Amrithasuresh one pull request covering ``Bio/Phylo/`` sounds good. Looking at the specific example:

https://github.com/biopython/biopython/blob/biopython-170/Bio/Phylo/TreeConstruction.py#L458

```python
    def _pairwise(self, seq1, seq2):
        """Calculate pairwise distance from two sequences.

        Returns a value between 0 (identical sequences) and 1 (completely
        different, or seq1 is an empty string.)
        """
        score = 0
```

Would become:

```python
    def _pairwise(self, seq1, seq2):
        """Calculate pairwise distance from two sequences (PRIVATE).

        Returns a value between 0 (identical sequences) and 1 (completely
        different, or seq1 is an empty string.)
        """
        score = 0
```

Note we keep that blank line, see PEP257 for background. Thank you!

**Comment by Amrithasuresh** [comment]
@peterjc Shall I do `Bio/GenBank/` folder as per the suggestions? Thank you.

**Comment by peterjc** [comment]
Sure - please make a new pull request for the ``Bio/GenBank/`` changes

**Comment by Amrithasuresh** [comment]
@peterjc I just pushed in my biopython git repository it is automatically merged with yesterday's pull request. How to fix that?

**Comment by peterjc** [comment]
You probably need to make a new branch, starting from the official Biopython repository's master branch.

**Comment by Amrithasuresh** [comment]
@peterjc Shall I do `Bio/SearchIO/` directory? I will create new branch then I will proceed the changes. Is it ok to give single pull request for this directory changes? Thank you.

**Comment by peterjc** [comment]
@Amrithasuresh You've nearly finished based on the grep command I was using - impressive patience/dedication 👍

**Comment by svalqui** [comment]
I'll have a look at this.

**Comment by svalqui** [comment]
Hi @peterjc , I have opened a PR, it address the files found by the grep command, and few more I found, I think that covers the scope of this issue.
Regards,

**Comment by peterjc** [comment]
Closed via #1749.
